anime-archive/src/components/section/Section.js

```diff
@@ -2,16 +2,18 @@ import AnimeCard from "../../components/animeCard/AnimeCard.js";
 import "./Section.css";
 
 const Section = (props) => {
+  const sections = props.apiData.data.data;
+
+  // console.log(data[props.data.extension], "props in section");
```

Comment:
Before merging to production can we remove console logs

Reply:
Whoops I completely forgot about the console logs, haha. Nice catch.

Reply:
Cleaned any remaining console.logs that weren’t not necessary, I also updated a few small things like the query and the 'dot' notation inside the files to account for the query update.